scss/themes/_branded.scss

```diff
@@ -22,17 +22,11 @@
 	.o-topper__tags,
 	.o-topper__standfirst {
 		box-sizing: border-box;
-		padding-right: 100px;
-
-		@include oGridRespondTo(M) {
-			padding-right: 150px;
-		}
+		@include oTopperHeadshotSpacing();
 	}
 
 	.o-topper__headline {
```

Comment:
did this class get a `padding-right` elsewhere?

Reply:
No didn't seem to, has other styles but not padding-right

Reply:
So by using the mixin here you'll be applying a padding-right that wasn't there before (under viewport: M)?

Reply:
Ah sorry, see what you mean. I kinda thought it made sense to apply styles to both as well, but on further inspection think it needs to remain as-is. Will tweak..